opentech/apply/funds/templates/funds/tables/table.html

```diff
@@ -1,29 +1,73 @@
 {% extends 'django_tables2/table.html' %}
-{% load table_tags %}
+{% load django_tables2 table_tags review_tags %}
 
 {# example of how to extend the table template #}
 {% block table.tbody.row %}
-    {{ block.super }}
+    <tr {{ row.attrs.as_html }} data-record-id="{{ row.record.id }}">
+        {% for column, cell in row.items %}
+            <td {{ column.attrs.td.as_html }}>{% if column.localize == None %}{{ cell }}{% else %}{% if column.localize %}{{ cell|localize }}{% else %}{{ cell|unlocalize }}{% endif %}{% endif %}</td>
+        {% endfor %}
+    </tr>
```

Comment:
Would prefer to keep the standard tables mechanism. Use the `row_attrs` meta option to add `data-record-id`, Would add extra data attribute to the child rows but means we dont need to copy in the row templating.

Reply:
👍